portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/logging/CompanyLogging.testcase

```diff
@@ -22,75 +24,398 @@ definition {
+		}
+
+		task ("Then the logs of one company should not appear in the logs of the other") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "o/company-log");
+
+			var groupId1 = HeadlessPortalInstanceAPI._getPortalInstanceGroupId(companyId = "${companyId1}");
+			var groupId2 = HeadlessPortalInstanceAPI._getPortalInstanceGroupId(companyId = "${companyId2}");
+			var groupId1 = ${groupId1} - 2;
+			var groupId2 = ${groupId2} - 2;
 
-		// TODO LRQA-75859
+			Click(
+				key_virtualHost = "www.able.com",
+				locator1 = "CompanyLogs#COMPANY_LOG_VIEW");
 
+			SelectWindow.selectPopUp(locator1 = "null");
+
+			AssertElementPresent(
+				key_logContent = "Initializing com.liferay.site.initializer.welcome for group ${groupId1}",
+				locator1 = "CompanyLogs#COMPANY_LOG_CONTENT");
+
+			AssertElementNotPresent(
+				key_logContent = "Initializing com.liferay.site.initializer.welcome for group ${groupId2}",
+				locator1 = "CompanyLogs#COMPANY_LOG_CONTENT");
+
+			Close.closeWindow();
+
+			Click(
+				key_virtualHost = "www.baker.com",
+				locator1 = "CompanyLogs#COMPANY_LOG_VIEW");
+
+			SelectWindow.selectPopUp(locator1 = "null");
+
+			AssertElementPresent(
+				key_logContent = "Initializing com.liferay.site.initializer.welcome for group ${groupId2}",
+				locator1 = "CompanyLogs#COMPANY_LOG_CONTENT");
+
+			AssertElementNotPresent(
+				key_logContent = "Initializing com.liferay.site.initializer.welcome for group ${groupId1}",
+				locator1 = "CompanyLogs#COMPANY_LOG_CONTENT");
+
+			Close.closeWindow();
+		}
 	}
 
 	@description = "If company.log.enabled=false, there should not be additional logging files created for each company and the UI access point should be unavailable"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CompanyLoggingDisabled {
+		property database.partition.enabled = "false";
+
+		task ("When the user tries navigating to the company logging UI access point") {
+			var baseURL = PropsUtil.get("portal.url");
+
+			Open.openNoError(locator1 = "${baseURL}/o/company-log");
+		}
+
+		task ("Then the page cannot be accessed") {
+			AssertLocation.assertPartialLocation(value1 = "web/guest");
+		}
+
+		task ("When the user adds a virtual instance") {
+			HeadlessPortalInstanceAPI.addPortalInstance(
+				domain = "www.able.com",
+				portalInstanceId = "www.able.com",
+				virtualHost = "www.able.com");
+		}
+
+		task ("Then additional company logging files should not be created") {
+			var logsDir = PropsUtil.get("liferay.home.dir.name");
 
-		// TODO LRQA-75853
+			var logsDir = "${logsDir}/logs";
 
+			var companiesFolderExists = FileUtil.exists("${logsDir}/companies");
+
+			if ("${companiesFolderExists}" == "true") {
+				fail("logs/companies folder exists");
+			}
+		}
 	}
 
 	@description = "An Omniadmin can get log names from all companies and download logs from all companies though the UI"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test GetAllCompanyLogs {
+		task ("Given two new virtual instances are created") {
```

Comment:
Same feedback as the other comment related to the virtual instance setup, remove un-necessary steps and utilize the backend when available to retrieve data to make the setup more robust and quick